ddtrace/contrib/pytest/plugin.py

```diff
@@ -11,398 +11,22 @@
-from ddtrace.contrib.coverage.patch import run_coverage_report
-from ddtrace.contrib.coverage.patch import unpatch as unpatch_coverage
-from ddtrace.contrib.coverage.utils import _is_coverage_invoked_by_coverage_run
-from ddtrace.contrib.coverage.utils import _is_coverage_patched
-from ddtrace.contrib.pytest.constants import DDTRACE_HELP_MSG
-from ddtrace.contrib.pytest.constants import DDTRACE_INCLUDE_CLASS_HELP_MSG
-from ddtrace.contrib.pytest.constants import FRAMEWORK
-from ddtrace.contrib.pytest.constants import KIND
-from ddtrace.contrib.pytest.constants import NO_DDTRACE_HELP_MSG
-from ddtrace.contrib.pytest.constants import XFAIL_REASON
-from ddtrace.contrib.unittest import unpatch as unpatch_unittest
-from ddtrace.ext import SpanTypes
-from ddtrace.ext import test
-from ddtrace.internal.ci_visibility import CIVisibility as _CIVisibility
-from ddtrace.internal.ci_visibility.constants import EVENT_TYPE as _EVENT_TYPE
-from ddtrace.internal.ci_visibility.constants import ITR_CORRELATION_ID_TAG_NAME
-from ddtrace.internal.ci_visibility.constants import ITR_UNSKIPPABLE_REASON
-from ddtrace.internal.ci_visibility.constants import MODULE_ID as _MODULE_ID
-from ddtrace.internal.ci_visibility.constants import MODULE_TYPE as _MODULE_TYPE
-from ddtrace.internal.ci_visibility.constants import SESSION_ID as _SESSION_ID
-from ddtrace.internal.ci_visibility.constants import SESSION_TYPE as _SESSION_TYPE
-from ddtrace.internal.ci_visibility.constants import SKIPPED_BY_ITR_REASON
-from ddtrace.internal.ci_visibility.constants import SUITE
-from ddtrace.internal.ci_visibility.constants import SUITE_ID as _SUITE_ID
-from ddtrace.internal.ci_visibility.constants import SUITE_TYPE as _SUITE_TYPE
-from ddtrace.internal.ci_visibility.constants import TEST
-from ddtrace.internal.ci_visibility.coverage import _module_has_dd_coverage_enabled
-from ddtrace.internal.ci_visibility.coverage import _report_coverage_to_span
-from ddtrace.internal.ci_visibility.coverage import _start_coverage
-from ddtrace.internal.ci_visibility.coverage import _stop_coverage
-from ddtrace.internal.ci_visibility.coverage import _switch_coverage_context
-from ddtrace.internal.ci_visibility.telemetry.constants import TEST_FRAMEWORKS
-from ddtrace.internal.ci_visibility.utils import _add_pct_covered_to_span
-from ddtrace.internal.ci_visibility.utils import _add_start_end_source_file_path_data_to_span
-from ddtrace.internal.ci_visibility.utils import _generate_fully_qualified_module_name
-from ddtrace.internal.ci_visibility.utils import _generate_fully_qualified_test_name
-from ddtrace.internal.ci_visibility.utils import get_relative_or_absolute_path_for_path
-from ddtrace.internal.ci_visibility.utils import take_over_logger_stream_handler
-from ddtrace.internal.constants import COMPONENT
-from ddtrace.internal.logger import get_logger
-
 
+DDTRACE_HELP_MSG = "HELP DDTRACE"
+DDTRACE_INCLUDE_CLASS_HELP_MSG = "HELP INCLUDE DDTRACE"
+NO_DDTRACE_HELP_MSG = "NO HELP"
```

Comment:
:question: what are these used for?

Reply:
OMG. Thank you for catching this. 😱 

They used to be imported from another module and I temporary declared them locally to avoid the import and make sure it worked, but never gave them their original values.